<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Random Test"
           tests="1"
           failures="1"
           errors="0"
           skipped="0"
           time="0.001">
  <properties />
            <testcase name="checkVacation" time="0.001" classname="testCases.Vacation">
                <failure type="utils.FrameworkException"
                  message="(null)"
              >
<![CDATA[Framework Exception:  Invalid Locator found
  utils.ObjectMethods.getElement(ObjectMethods.java:63)
  utils.ObjectMethods.clickElement(ObjectMethods.java:16)
  testCases.Vacation.checkVacation(Vacation.java:24)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
  org.testng.TestRunner.privateRun(TestRunner.java:774)
  org.testng.TestRunner.run(TestRunner.java:624)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
  org.testng.SuiteRunner.run(SuiteRunner.java:261)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
  org.testng.TestNG.run(TestNG.java:1024)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:74)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:92)
  org.apache.maven.surefire.Surefire.run(Surefire.java:177)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:338)
  org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:997)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
